acme_diags/parameter/meridional_mean_2d_parameter.py

```diff
@@ -1,11 +1,32 @@
 import numpy
 from .core_parameter import CoreParameter
-
+from acme_diags.driver.utils.general import strictly_increasing,strictly_decreasing,non_increasing,non_decreasing,monotonic
 
 class MeridionalMean2dParameter(CoreParameter):
     def __init__(self):
         super(MeridionalMean2dParameter, self).__init__()
-        self.plevs = numpy.logspace(2.0, 3.0, num=17)
+        self.plevs = numpy.logspace(2.0, 3.0, num=17).tolist()
+        self.plot_log_plevs = False
+        self.plot_plevs = False
         # Granulating plevs causes duplicate plots in this case.
-        # So keep all of the default values except plevs. 
+        # So keep all of the default values except plevs.
         self.granulate.remove('plevs')
+
+    def check_values(self):
+        plevs = self.plevs
+        if not isinstance(plevs,list):
+           msg = 'plevs needs to be a list'
+           raise RuntimeError(msg)
+
+        if len(plevs) > 1:
+            if monotonic(plevs):
+                if plevs[0] > plevs[1]:
+                    plevs = plevs[::-1]
```

Comment:
What are you trying to accomplish here? If `plevs[0] > plevs[1]` is `True` then `plevs` must be montonically decreasing. If so, then you reverse the entire list. This would make it monotonically increasing. Why don't you just check that plevs is`montonically_increasing` in the first place then?

Reply:
`monotonic` is either decreasing or increasing. I need to make sure the list is monotonic first.

Reply:
This is true, but then you effectively force `plevs` to be monotonically increasing, so why not just check that? You have the function for it in `utils.general`? Are users using montonically decreasing `plevs` so frequently that we'll just reverse the list for them?

Reply:
it's not obvious that what order users will put in for plevs. I would say 50-50 of chance.